tools/eslint-rules/inspector-check.js

```diff
@@ -31,7 +31,16 @@ module.exports = function(context) {
 
   function reportIfMissing(context, node) {
     if (usesInspector && !hasInspectorCheck) {
-      context.report(node, msg);
+      context.report({
+        node,
+        message: msg,
+        fix: (fixer) => {
+          return fixer.insertTextAfter(
+            node,
+            '\n\n common.skipIfInspectorDisabled();'
```

Comment:
What does the `node` refer to in this context? I feel like this might be getting inserted in the wrong place.

Also, why the errant space after `\n\n`?

Reply:
You are right. This inserts the correction at the end of the file. Working on inserting it below ```require('../common')```. Also will fix space after```\n\n```.

Reply:
Made a new commit fixing the same. Please verify. Thanks.